armi/reactor/components/component.py

```diff
@@ -757,6 +787,10 @@ def setDimension(self, key, val, retainLink=False, cold=True):
         """
         Set a single dimension on the component.
 
+        .. impl:: Set a dimension of a component
+            :id: I_ARMI_COMP_VOL0
+            :implements: R_ARMI_COMP_VOL
+
```

Comment:
Remove this.

This belongs on the methods `getArea()` and `getVolume()` in this class (Component).